builtin/logical/mysql/backend_test.go

```diff
@@ -220,14 +252,26 @@ func testAccStepConfig(t *testing.T, d map[string]interface{}, expectError bool)
 	}
 }
 
-func testAccStepRole(t *testing.T) logicaltest.TestStep {
+func testAccStepRole(t *testing.T, wildCard bool) logicaltest.TestStep {
+
+	pathData := make(map[string]interface{})
+	if wildCard == true {
+		pathData = map[string]interface{}{
+			"sql": testRoleWildCard,
+		}
+	} else {
+		pathData = map[string]interface{}{
```

Comment:
Any reason not to test with `testRoleHost` with and without `testRevokeSQL`? Any reason not to test `testRevokeSQL` with `testRoleWildCard`?

Adding a second bool here would allow all four possibilities.